SunEngine.Commons/Controllers/PulseController.cs

```diff
@@ -13,37 +14,60 @@ namespace SunEngine.Commons.Controllers
     public class PulseController : BaseController
     {
         protected readonly DataBaseConnection db;
-        
+
         public PulseController(
             DataBaseConnection db,
             IServiceProvider serviceProvider) : base(serviceProvider)
         {
             this.db = db;
         }
-        
+
         [HttpGet]
         [HttpPost]
         [AllowAnonymous]
         public virtual IActionResult Pulse()
         {
             return Ok(new {SomeDataTest = "SomeData Test!"});
         }
-        
+
         [HttpGet]
         [HttpPost]
         [AllowAnonymous]
         public virtual async Task<IActionResult> PulseDb()
         {
-            bool anyUserGroup = await  db.Roles.AnyAsync();
+            bool anyUserGroup = await db.Roles.AnyAsync();
             return Ok(new {db_UserGroups_AnyAsync = anyUserGroup});
```

Comment:
Could you clarify this code for me? what db_UserGroups_AnyAsync is? I'm confused with the naming of it - is it a conventional name?

Reply:
This is old name befor userGroup => Role Refacter. I will change it. Thanks.